test/acceptance/scratch/attribute.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Test the attributes of a given element
```

Comment:
not sure if we need the whole scratch folder yet - probably can delete/refactor to only what we need

Reply:
The scratch folder is not used anywhere as of now, so it can all be deleted. Also, I don't think the support folder is used anywhere so can this be deleted as well?

Reply:
Agree - @json-hamilton  - lets check with @slimjeo3251  as well